src/main/scala/org/allenai/plugins/VersionInjectorPlugin.scala

```diff
@@ -117,6 +130,34 @@ object VersionInjectorPlugin extends AutoPlugin {
         gitConfFile
     }
 
+  val stageChecksumTask = stageChecksum := {
+    import java.io.{ File => JFile }
```

Comment:
I believe SBT has implicit conversions from its file to Java files. I don't *think* you should need this.